examples/low-level-api/list-offers.py

```diff
@@ -41,7 +41,7 @@ def main():
     subnet = args.subnet_tag
     sys.stderr.write(f"Using subnet: {utils.TEXT_COLOR_YELLOW}{subnet}{utils.TEXT_COLOR_DEFAULT}\n")
 
-    enable_default_logger(level=args.log_level)
```

Comment:
why?

Reply:
Because `enable_default_logger` doesn't have `level` argument (any more, it used to have it) so this call won't work.